apps/aecore/lib/aecore/account/tx/spend_tx.ex

```diff
@@ -121,10 +119,8 @@ defmodule Aecore.Account.Tx.SpendTx do
         %{},
         block_height,
         %SpendTx{amount: amount, receiver: %Identifier{value: receiver}},
-        %DataTx{} = data_tx
+        %DataTx{senders: [%Identifier{value: sender}]}
```

Comment:
I assume this fails if multiple senders are present

Reply:
Number of senders shuld be checked in `validate` and it's enforced by encoding